notify/receiver.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package notify
+
+type Receiver struct {
+	name         string
```

Comment:
It looks like there is already a receiver in nflog and it has field GroupName, which is the same as this structure's field `name`. Although I think the name is confusing, I would prefer to be consistent across models.

Reply:
Interesting! I found another model `Receiver` in API models and it has field `Name`. So keeping this name is fine.